modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-restricted-network_{context}"]
+= Enabling your Operator for restricted network environments
+
+As an Operator author, your CSV must meet the following additional requirements
+for your Operator to run properly in a restricted network environment:
+
+* List any _related images_, or other container images that your Operator might
+require to perform their functions.
+* Reference all specified images by a digest (SHA) and not by a tag.
+
+.Prerequisites
+
+* An Operator project with a CSV
+
+.Procedure
+
+* In your Operator's CSV, define a list of any related images:
++
+[source,yaml]
+----
+kind: ClusterServiceVersion
+metadata:
+  name: etcd-operator
+spec:
+[...]
+  relatedImages: <1>
+  - name: default
+    image: quay.io/coreos/etcd@sha256:12345 <2>
+    annotation: default
```

Comment:
we didn't implement the annotation part, that's part of the "future work" from the proposal

Reply:
Removed.